ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.collection.view.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ /*global require*/
+ const Marionette = require('marionette');
+ const _ = require('underscore');
+ const $ = require('jquery');
+ const SearchFormView = require('component/search-form/search-form.view');
+ const ResultFormCollection = require('./result-form.collection');
+ const CustomElements = require('js/CustomElements');
+
+ module.exports = Marionette.CollectionView.extend({
+     childView: SearchFormView,
+     className: 'is-list is-inline has-list-highlighting',
+     tagName: CustomElements.register('result-forms'),
+     initialize: function(options) {
+        let resultFormCollection = new ResultFormCollection();
```

Comment:
Lets declare this as a constant outside the `CollectionView`, because re-instantiating it on every call that renders the dropdown will create transient instances of the child model. This becomes a problem when re-populating the lightbox with the security permissions for each particular metacard as well as anything that relies on the model state that is nested within this collection.